buildSrc/src/main/java/net/neoforged/neodev/utils/structure/ClassInfo.java

```diff
@@ -0,0 +1,27 @@
+package net.neoforged.neodev.utils.structure;
+
+import org.apache.commons.lang3.mutable.MutableInt;
+import org.gradle.api.Named;
+
+import java.util.List;
+
+public record ClassInfo(String name, MutableInt access, List<ClassInfo> parents, List<MethodInfo> methods,
+                        List<FieldInfo> fields) implements Named {
```

Comment:
Why implement Named? You can just use `.name()` on the record.

Reply:
For the `named` predicate to work.